docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -0,0 +1,220 @@
+
+:::warning Experimental features ahead
+
+At the time of writing this guide (December 2023), request locking is still an experimental feature. You can read more about the experiment by visiting the [request locking experiment](../experiments/experiments-request-locking) page.
+
+:::
+
+In this guide, we will walk through how you can turn your single scraper into a scraper that can be parallelized and run in multiple instances. This guide
+assumes you've read and walked through our [introduction guide](../introduction/setting-up) (or have a fully-fledged scraper already built), but if you haven't done so yet,
+take a break, go read through all that, and come back. We'll be waiting...
+
+*Oh, you're back already! Let's proceed in making that scraper parallel!*
+
+## Things to consider before parallelizing
+
+Before you rush ahead and change your scraper to support parallelization, take a minute to consider the following factors:
+
+- do you plan on scraping so many pages that you need to parallelize your scraper?
+  > for example, if your scraper goes across few pages, you probably don't need parallelization  
+  > but if you scrape a lot of pages, or you scrape pages that take a long time to load, you might want to consider parallelization
+- can you parallelize your scraper while not overloading the source website?
+  > for example, if you scrape a website that has a lot of traffic, you don't want to add to that traffic by running multiple scrapers in parallel
+  > as that might cause the website to go down for all its users
+- do you have the resources available to run multiple scrapers in parallel?
+  > when running locally, depending on your scraper type, do you have enough CPU and RAM available to sustain multiple scrapers running in parallel  
+  > when running in the cloud, will the extra speed from parallelization be worth the extra cost of running multiple scrapers in parallel?
+
+Let's assume you answered yes to all of those. Yes? Yes. Before we go ahead and get to the actual guide, we'd like to ask you to also take a read on [Apify's Ethical Web Scraping](https://blog.apify.com/what-is-ethical-web-scraping-and-how-do-you-do-it/)
+blog post!
+
+Now that we've gone through all that, the guide is split into two parts: converting the initial scraper we built in the [introduction guide](../introduction/setting-up) to one that prepares requests
+to be usable in parallel scrapers, and then running scrapers in parallel.
+
+:::note Want to see the final result?
+
+You can see it on the [Crawlee Parallel Scraping Example](https://github.com/apify/crawlee-parallel-scraping-example) repository! It's the same scraper we built in the [introduction guide](../introduction/setting-up), but in TypeScript and
+parallelized!
+
+:::
+
+### But isn't Crawlee already concurrent? What's the difference between concurrency and parallelization?
+
+> *Hold on! I've used Crawlee before, and it has a `maxConcurrency` option on it! What's this for then?!*
```

Comment:
If using schemas can be considered, I think it's easier for the reader to visually understand the concepts.

> This what crawlee is doing by defaut (concurrency):
> ![image](https://github.com/apify/crawlee/assets/1524501/6008a2a7-3eed-444a-9d05-d95876fb10d0)
> This what we're achieving in this guide (parallelism):
> ![image](https://github.com/apify/crawlee/assets/1524501/6fcd534e-cefe-4c9c-8a25-29f3904907e1)